paper.md

```diff
@@ -220,8 +221,7 @@ Such annotation is not sufficient to introduce changes by following the descript
   either will not be applicable or just would miss some Xs if applied to a vastly different version -->
 
 
-
-**They are generic and lack support for domain-specific solutions.**
+**Domain-specific solutions.**
```

Comment:
I'd vote for something like @mih proposed in his comment: "additional software adaptors" rather than suggesting we add "domain-specific" solutions. We explicitly want datalad to not be domain specific, so this wording feels contradictory.

Reply:
FWIW - those are not necessarily "software" but could be "metadata" adapters etc.
What about "Targeted interfaces and interoperability adapters" as worded in the summary ATM?

Reply:
Yes, fine with me.

Reply:
committed